main.js

```diff
@@ -187,79 +181,72 @@ function showTimer() {
 function clockFormat() {
   var totalSec = parseInt(currentActivity.seconds);
   var totalMin = parseInt(currentActivity.minutes * 60);
-  var time = totalSec + totalMin
-
+  var time = totalSec + totalMin;
```

Comment:
This whole function should be in your activity class since that is what the instructors gave that to you in this repo. Move it and figure out what arguments/parameters you will need